lib/rules/color-named/index.js

```diff
@@ -49,7 +50,7 @@ const rule = function (expectation, options) {
     const namedColors = Object.keys(namedColorData)
 
     root.walkDecls(decl => {
-      if (propertySets.acceptCustomIdents.has(decl.prop)) {
+      if (propertySets.acceptCustomIdents.has(decl.prop) || !isStandardSyntaxDeclaration(decl)) {
```

Comment:
@jeddy3 so far pretty straight forward, but the maps return `false` for the `isStandardSyntaxFunction` as maps aren't given a value by `postcss-value-parser`. Thoughts?